### PR TITLE
Transactions in localstorage

### PR DESCRIPTION
**Pull request by raphab3**
## Description
Added: Save/Delete of transactions in localstorage

image

### CODE REVIEW

File: assets/js/principal.js [hunk 1]
```diff
@@ -2,29 +2,22 @@ const form = document.querySelector('form');
...
 
 function main() {
   handleSelectType('entry');
-}
-
-const getTransations = () => {
-  const data = window.localStorage.getItem('transactions');
-  return JSON.parse(data);
-};
+  console.log('transactions: ', transactions);
```
Comment (raphab3) [inline assets/js/principal.js:11]: remova esse console.log

File: assets/js/principal.js [hunk 2]
```diff
@@ -2,29 +2,22 @@ const form = document.querySelector('form');
...
+  generateTrs(transactions);
+}
 
-const loadTable = () => {
-  for (const transaction of transactions) {
-    console.log('tra', transaction);
-  }
+const getLastCountTransactions = () => {
+  countTransactions = getData('count-transactions') || 1;
+  console.log('countTransactions: get: ', countTransactions);
```
Comment (raphab3) [inline assets/js/principal.js:19]: remover

File: assets/js/principal.js [hunk 3]
```diff
@@ -66,54 +59,32 @@ const handleSubmit = (e) => {
...
-
-    if (value === 'DESPESA') {
-      td.classList.add('badge2');
-      td.classList.add('bg-danger');
-    }
-
-    td.textContent = value;
-    tr.appendChild(td);
-  });
+  console.log('transaction: ', transaction);
```
Comment (raphab3) [inline assets/js/principal.js:68]: remover

**Review by marcos7junior — APPROVED** [review]
Very good!

**Pull request merged**